hyo2/soundspeed/formats/readers/valeport.py

```diff
@@ -378,34 +394,83 @@ def _mini_body(self):
         self.ssp.cur.data_resize(count)
 
     def _midas_body(self):
+        # check valid format
+        first_row = self.lines[self.samples_offset]
+        first_row = first_row.split('\t')
+        if not first_row[0].strip().upper() == 'DATE / TIME':
+            raise RuntimeError('Invalid first data roq: %s' % first_row)
+
+        logger.debug('midas format: body')
+
+        pressure_idx = None     # Pressure, Speed, and Temperature in all probe types
```

Comment:
The `# Pressure, Speed, and Temperature in all probe types` comment is derived on observation of the current test files.
It would be better to test if we have found the indices for pressure, speed, and temperature. In case not, throw a meaningful error for ourselves in the coming future!